database/database_functions.py

```diff
@@ -1,64 +1,96 @@
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-from database_models import Persons, Rooms, Allocations, BASE
+from database.database_models import Persons, Rooms, BASE
 
-ENGINE = create_engine('sqlite:///amity.db')
 
-BASE.metadata.create_all(ENGINE)
-DBSESSION = sessionmaker(bind=ENGINE)
-SESSIONS = DBSESSION()
 
-def get_persons():
-    persons = SESSIONS.query(Persons).all()
-    p_dct = {}
-    for person in persons:
-        p_dct.update({person.p_name:person.p_type})
+class AmityDatabase(object):
 
-    return p_dct
+    '''
+    class contains function for the database
+    '''
 
-def get_rooms():
-    rooms = SESSIONS.query(Rooms).all()
-    r_dct = {}
-    for room in rooms:
-        r_dct.update({room.r_name:room.r_type})
+    def __init__(self):
+        pass
 
-    return r_dct
+    def initialize(self, filename):
```

Comment:
Why not have the functions of this function be under `__init__()` rather than creating a different function called `initialize`?